third_party/nvfuser/csrc/multidevice/multidevice_runtime.cpp

```diff
@@ -78,7 +78,8 @@ MultiDeviceRuntime::CompiledKernelPtr MultiDeviceRuntime::compileCluster(
 
   args.setDeviceIndex(device_index);
   // Lower the fusion and compile the generated kernel.
-  executor_ptr->compileFusion(fusion_from_cluster.get(), args, launch_params);
+  executor_ptr->compileFusion(
+      fusion_from_cluster.get(), args, launch_params, {});
```

Comment:
This is another compilation error:
```C++
/home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/multidevice/multidevice_runtime.cpp:81:30: error: no matching function for call to ‘nvfuser::FusionExecutor::compileFusion(std::unique_ptr<nvfuser::Fusion>::pointer, nvfuser::KernelArgumentHolder&, nvfuser::LaunchParams&)’
   81 |   executor_ptr->compileFusion(fusion_from_cluster.get(), args, launch_params);
      |   ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/kernel_cache.h:4,
                 from /home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/fusion_segmenter.h:5,
                 from /home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/multidevice/multidevice_runtime.cpp:2:
/home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/executor.h:62:8: note: candidate: ‘void nvfuser::FusionExecutor::compileFusion(nvfuser::Fusion*, const nvfuser::KernelArgumentHolder&, const nvfuser::LaunchParams&, nvfuser::CompileParams)’
   62 |   void compileFusion(
      |        ^~~~~~~~~~~~~
/home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/executor.h:62:8: note:   candidate expects 4 arguments, 3 provided
/home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/executor.h:71:8: note: candidate: ‘void nvfuser::FusionExecutor::compileFusion(nvfuser::Fusion*, const c10::ArrayRef<c10::IValue>&, const nvfuser::LaunchParams&, nvfuser::CompileParams)’
   71 |   void compileFusion(
      |        ^~~~~~~~~~~~~
/home/gaoxiang/nvfuser7/third_party/nvfuser/csrc/executor.h:73:40: note:   no known conversion for argument 2 from ‘nvfuser::KernelArgumentHolder’ to ‘const c10::ArrayRef<c10::IValue>&’
   73 |       const at::ArrayRef<c10::IValue>& inputs = {},
```

Reply:
Thanks. I guess I didn't see it as I had `USE_DISTRIBUTED=0`.